### DISCUSSION TITLE
Upgrade to ESLint 9.x?

### DISCUSSION THREAD

**Discussion by MBurchard** [antfu/eslint-config#561] [General]
Hi @antfu 
are there any plans to upgrade the ESLint dependency to 9.x (9.7 as of today)
Best
Martin

**Comment by Dimava** [comment]
@MBurchard it is already supported for quite a while, but migration to it is a breaking change , so it'll become default only in 3.0
https://github.com/antfu/eslint-config/issues/558#issuecomment-2241782319
